metecho/api/serializers.py

```diff
@@ -32,6 +32,14 @@
 User = get_user_model()
 
 
+def get_assigned_github_user(project_id, assigned_id):
+    project = Project.objects.get(id=project_id)
+    assigned_gh_user = list(
+        filter(lambda x: x["id"] == assigned_id, project.github_users)
+    )
+    return assigned_gh_user[0] if assigned_gh_user else ""
```

Comment:
This raises an exception:

```
  File "/app/metecho/api/serializers.py", line 719, in get_assigned_dev
    return get_assigned_github_user(
  File "/app/metecho/api/serializers.py", line 38, in get_assigned_github_user
    filter(lambda x: x["id"] == assigned_id, project.github_users)
TypeError: 'ManyRelatedManager' object is not iterable
```

Reply:
Fixed in 9c2f3f8fdd1c41b649bb57845d02a532b760ee6a